### PR TITLE
[py3] took out all tests of python version

### PR DESCRIPTION
**Pull request by lasofivec**
Took out all lingering `if sys.version[0]=='2':`

### CODE REVIEW

File: tofu/geom/_comp.py [hunk 1]
```diff
@@ -11,11 +11,6 @@
 import scipy.interpolate as scpinterp
 import scipy.integrate as scpintg
 
-if sys.version[0] == "3":
-    from inspect import signature as insp
```
Comment (Didou09) [inline tofu/geom/_comp.py]: If this is suppressed too, where is insp defined ?

File: tofu/geom/_comp.py [hunk 2]
```diff
@@ -1027,18 +1022,15 @@ def LOS_calc_signal(
         D, u, dL, DL=DL, dLMode=dLMode, method=method, Test=Test
     )
     out = insp(ff)
```
Comment (Didou09) [inline tofu/geom/_comp.py]: insp is called here

Comment (lasofivec) [reply tofu/geom/_comp.py]: sorry ! you are right ! fixed

File: tofu/geom/_comp.py [hunk 3]
```diff
@@ -10,11 +10,7 @@
 import numpy as np
 import scipy.interpolate as scpinterp
 import scipy.integrate as scpintg
-
-if sys.version[0] == "3":
-    from inspect import signature as insp
-elif sys.version[0] == "2":
-    from inspect import getargspec as insp
+from inspect import signature as insp
```
Comment (lasofivec) [inline tofu/geom/_comp.py]: corrected !

**Pull request merged**